app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -157,7 +157,8 @@ def save_container_groups_inventory(ems, hashes)
       h[:container_node_id]       = h.fetch_path(:container_node, :id)
       h[:container_replicator_id] = h.fetch_path(:container_replicator, :id)
       h[:container_project_id]    = h.fetch_path(:project, :id)
-      h[:container_build_pod_id]  = ems.container_build_pods.find_by(:name => h[:build_pod_name]).try(:id)
+      h[:container_build_pod_id]  = ems.container_build_pods.find_by(:name      => h[:build_pod_name],
+                                                                     :namespace => h.fetch_path(:project, :name)).try(:id)
```

Comment:
It's weird that we have a `namespace` string and not `container_project_id` like everywhere else. We should probably fix that